cmd/golangBuild.go

```diff
@@ -570,7 +570,7 @@ func lookupGolangPrivateModulesRepositories(goModFile *modfile.File, globPattern
 }
 
 func runBOMCreation(utils golangBuildUtils, outputFilename string) error {
-	if err := utils.RunExecutable("cyclonedx-gomod", "mod", "-licenses", "-test", "-output", outputFilename); err != nil {
+	if err := utils.RunExecutable("cyclonedx-gomod", "mod", "-licenses", "-test", "-output", outputFilename, "-ouput-version", "1.4"); err != nil {
```

Comment:
I see a typo here ouput-version
It should be "output-version"

@Jk1484 , could you please also test changes before marking it for review?

Reply:
> could you please also test changes before marking it for review?

That would be great since there are people using this :(